quantecon/markov/approximation.py

```diff
@@ -212,3 +214,200 @@ def _fill_tauchen(x, P, n, rho, sigma, half_step):
             z = x[j] - rho * x[i]
             P[i, j] = (std_norm_cdf((z + half_step) / sigma) -
                        std_norm_cdf((z - half_step) / sigma))
+
+
+def cartesian_product(arrays, row_major=True):
```

Comment:
Does this do something different from [`quantecon.cartesian`](https://github.com/QuantEcon/QuantEcon.py/blob/89a70127b1321444b8f22ba262c7401f4b08e9c4/quantecon/gridtools.py#L13)?